consensus/src/runway/mod.rs

```diff
@@ -1099,3 +1147,26 @@ pub(crate) async fn run<H, D, US, UL, MK, DP, FH, SH>(
 
     debug!(target: "AlephBFT-runway", "{:?} Runway ended.", index);
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::runway::format_missing_coords;
+
+    #[test]
+    pub fn formats_missing_coords() {
+        assert_eq!(format_missing_coords(&[]), "");
+        assert_eq!(format_missing_coords(&[(0, 13)]), "{Creator 0: 13}");
+        assert_eq!(
+            format_missing_coords(&[(0, 1), (0, 2)]),
+            "{Creator 0: [1-2]}"
+        );
+        assert_eq!(
+            format_missing_coords(&[(0, 1), (0, 3), (0, 4), (0, 5), (0, 6), (0, 9)]),
+            "{Creator 0: 1,[3-6],9}"
+        );
+        assert_eq!(
+            format_missing_coords(&[(1, 3), (0, 1), (1, 1), (3, 0)]),
```

Comment:
That's a bit _surprising_ , isn't it? Why is `(1, 3)` before `(0, 1)` ? This broke just now? `(1,3)` should follow `(1,1)` 🤔

Reply:
It isn't - I just permuted input coords, cause now they may be not sorted, and the expected output string hasn't changed :stuck_out_tongue: